.gitignore

```diff
@@ -0,0 +1,106 @@
+# Logs
```

Comment:
Will the following suffice?:

```
node_modules
*.log
```

Reply:
Yup; I used the default provided by GitHub, happy to slim it down. I'll merge your and @Mouvedia's suggestion.